usaspending_api/etl/es_transaction_template.json

```diff
@@ -82,7 +82,7 @@
         },
         "award_category": {
           "type": "text",
-          "index": true
+          "fielddata": true
```

Comment:
Does this need to use fielddata? did you try with a `keyword` type? (since this has high cardinality)

Reply:
I know, I tried it, and it won't work without it. I tried both keyword and index=true. If you want to aggregate text fielddata: true. It was the trade-off I selected instead of doing 5-7 http requests for one query.